libraries/AP_Math/control.cpp

```diff
@@ -210,6 +210,17 @@ void shape_accel_xy(const Vector2f& accel_input, Vector2f& accel,
     }
 }
 
+void shape_accel_xy(const Vector3f& accel_input, Vector3f& accel,
```

Comment:
we should probably have a little comment above this function like we have for the others so it doesn't feel left out..

Reply:
Ah, actually it's ok.. the function above it is the same and has lots of comments. txs

Reply:
It is left out, sometimes honesty is the best policy.